### ISSUE TITLE
panic running the test

### ISSUE THREAD

**Issue by mator** [rogerwelin/cassowary#18]
centos-7 x86_64 VM
````
$ cassowary/cassowary -v
cassowary - 學名 version 0.9.0
$ cassowary/cassowary run -u https://dev.test -c 500 -n 100000 --disable-keep-alive

Starting Load Test with 100000 requests using 500 concurrent users

  14% |█████                                   | [44s:4m36s]            panic: Get "https://dev.test": context deadline exceeded (Client.Timeout exceeded while awaiting headers)

goroutine 262 [running]:
github.com/rogerwelin/cassowary/pkg/client.(*Cassowary).runLoadTest(0xc0002dc000, 0xc000300000, 0xc00009c060)
        /Users/roger/programmering/cassowary/pkg/client/load.go:93 +0x144a
github.com/rogerwelin/cassowary/pkg/client.(*Cassowary).Coordinate.func1(0xc0002dc000, 0xc000300000, 0xc00009c060, 0xc000022630)
        /Users/roger/programmering/cassowary/pkg/client/load.go:180 +0x3f
created by github.com/rogerwelin/cassowary/pkg/client.(*Cassowary).Coordinate
        /Users/roger/programmering/cassowary/pkg/client/load.go:179 +0x2b6````

**Comment by rogerwelin** [comment]
Hi @mator, thanks for raising the issue.
The http requests failes, probably the server is overloaded hence the timeout, so Cassowary cannot do much more than exit what it's doing. However the panic is not that pretty or user-friendly, I will make a releaser later today to more gracefully handle this error

**Comment by rogerwelin** [comment]
Fixed in v0.9.1
